extensions/transform/org.eclipse.smarthome.transform.scale/src/main/java/org/eclipse/smarthome/transform/scale/internal/ScaleTransformationService.java

```diff
@@ -47,9 +47,6 @@
      *            the list of properties defining all the available ranges
      * @param source
      *            the input to transform
-     *
```

Comment:
This JavaDoc seems to be wrong formatted for me at all.

So, not really part of your "remove inheritDoc" changes, but I don't think a correct JavaDoc starts with a `<p>` at all.
Shouldn't it start with a short sentence and then perhaps a longer description.

Another todo that we should remember is the removal of all `</p>`. AFAIK there is only `<p>` defined for "correct" JavaDoc.